compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -1606,16 +1606,16 @@ impl<'a, 'tcx> EncodeContext<'a, 'tcx> {
     }
 
     fn encode_info_for_anon_const(&mut self, id: hir::HirId) {
-        let def_id = self.tcx.hir().local_def_id(id);
-        debug!("EncodeContext::encode_info_for_anon_const({:?})", def_id);
-        let body_id = self.tcx.hir().body_owned_by(id);
+        let local_did = self.tcx.hir().local_def_id(id);
```

Comment:
I was also going to keep using `did` for pattern matching but based on your other comment, I'll use `def_id`. Should I use `def_id` instead of `local_did`?